docs/cloud/what-are-the-namespace-level-permissions-for-users-in-temporal-cloud.md

```diff
@@ -0,0 +1,15 @@
+---
+id: what-are-the-namespace-level-permissions-for-users-in-temporal-cloud
+title: What are the Namespace-level permissions for users in Temporal Cloud?
+sidebar_label: Namespace-level permissions
+description: Namespace-level permissions are Namespace Admin, Write, and Read-Only.
+---
+
+A [Global Admin](/cloud/what-are-the-account-level-roles-for-users-in-temporal-cloud) can assign permissions for any [Namespace](/namespaces) in an account.
+A Developer can assign permissions for a Namespace they create.
+
+For a Namespace, a user can have one of the following permissions:
+
+- **Namespace Admin:** Can create, rename, update, and delete [Workflows](/workflows) within the Namespace
```

Comment:
cannot "rename" a Workflow. "Delete" also doesn't make sense. Furthermore, I'm not sure we should call out permissions on workflows within a namespace, but only call out permissions on operations we can perform on the configuration of a namespace instead. Same applies to description in account level roles section

Reply:
I expanded the description from "CRUD," but I agree that it doesn't make sense in this context. I need to chase down the current distinction between Namespace Admin and Write permissions.